onnxruntime/core/providers/dml/DmlExecutionProvider/src/DmlGraphFusionHelper.cpp

```diff
@@ -132,8 +133,9 @@ namespace DmlGraphFusionHelper
 
         // Walk through each graph edge and mark used inputs
         inputsUsed.assign(fusedNodeInputCount, false);
-        for (const DML_INPUT_GRAPH_EDGE_DESC& edge : inputEdges) {
-            inputsUsed[edge.GraphInputIndex] = true;
+        for (uint32_t idx = 0; idx < inputEdgeCount; idx++) {
```

Comment:
```suggestion
        for (uint32_t index = 0; index < inputEdgeCount; ++index)
        {
```

whole words convention